src/main/scala/com/carta/compat/java/excel/AppendOnlySheetWriter.scala

```diff
@@ -0,0 +1,20 @@
+package com.carta.compat.java.excel
+
+
+import com.carta.compat.java.exscalabur.{DataCell, DataRow}
+
+import scala.collection.JavaConverters._
+
+class AppendOnlySheetWriter(scalaSheetWriter: com.carta.excel.AppendOnlySheetWriter) {
+  def writeStaticData(data: java.util.List[DataCell]): Unit = {
+    scalaSheetWriter.writeStaticData(data.asScala.map(_.asScala))
```

Comment:
can we have a test for these that just asserts it calls the method with the converted scala types?